sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.institutions.controller.saveScholasticStanding.e2e-spec.ts

```diff
@@ -707,6 +707,101 @@ describe("StudentScholasticStandingsInstitutionsController(e2e)-saveScholasticSt
       });
   });
 
+  it("Should not create new scholastic standing 'Student did not complete program' for a part-time application when number of unsuccessful completion weeks is greater than the total number of offering weeks.", async () => {
+    // Arrange
+    mockFormioDryRun({
+      studentScholasticStandingChangeType:
+        StudentScholasticStandingChangeType.StudentDidNotCompleteProgram,
+      numberOfUnsuccessfulWeeks: 50,
+    });
+    const application = await saveFakeApplication(
```

Comment:
No need to save the application to validate the DTO validations.
The endpoint can have a fake application ID as below.
```ts
const endpoint = `/institutions/scholastic-standing/location/${collegeFLocation.id}/application/99999`;
```